src/Faker/Provider/HtmlLorem.php

```diff
@@ -47,6 +47,10 @@ public function __construct(Generator $generator)
      */
     public function randomHtml($maxDepth = 4, $maxWidth = 4)
     {
+        if (!class_exists('DOMDocument', false)) {
+            return new \RuntimeException('DOMDocument is required to use randomHtml.');
```

Comment:
```suggestion
            throw new \RuntimeException('DOMDocument is required to use randomHtml.');
```

Can we also add this to the DocBlock, if we insist on throwing an exception here?

Reply:
Done, thanks!